### PR TITLE
TE 36 fix dependencies

### PR DESCRIPTION
**Pull request by stereomon**
Fixed missing dependencies.

### CODE REVIEW

**Comment by dereuromark** [comment]
Looks like there is a dependency issue: https://travis-ci.com/spryker/event-behavior/builds/84411112

**Comment by dereuromark** [comment]
The lock file mustr also be excluded for such standalone modules.
Same for the other to behavior ones.

File: composer.json [hunk 1]
```diff
@@ -26,8 +27,7 @@
   },
   "suggest": {
     "spryker/console": "*",
-    "spryker/silex": "^2.0.0",
-    "spryker/symfony": "^3.0.0"
+    "spryker/silex": "^2.0.0"
```
Comment (dereuromark) [inline composer.json]: This shouldnt be version numbers but reasons why (text).

**Comment by dereuromark** [comment]
Now the correct travis builds work: https://travis-ci.com/spryker/event-behavior/builds/84446070

File: composer.json (hunk 1 above)
Comment (stereomon) [reply composer.json]: I know, but why it is merged by you before I have the chance to fix that?

Comment (dereuromark) [reply composer.json]: you can always make a follow up.

**Pull request merged**